virtual_rainforest/models/soil/carbon.py

```diff
@@ -251,48 +269,74 @@ def calculate_binding_coefficient(
     return 10.0 ** (binding_with_ph_slope * pH + binding_with_ph_intercept)
 
 
-def convert_temperature_to_scalar(
-    soil_temp: NDArray[np.float32],
-    temp_scalar_coefficient_1: float,
-    temp_scalar_coefficient_2: float,
-    temp_scalar_coefficient_3: float,
-    temp_scalar_coefficient_4: float,
-    temp_scalar_reference_temp: float,
+def calculate_temperature_effect_on_microbes(
+    soil_temperature: NDArray[np.float32],
+    activation_energy: float,
+    reference_temperature: float,
+    gas_constant: float,
 ) -> NDArray[np.float32]:
-    """Convert soil temperature into a factor to multiply rates by.
+    """Calculate the effect that temperature has on microbial metabolic rates.
 
-    This form is used in :cite:t:`abramoff_millennial_2018` to minimise differences with
-    the CENTURY model. We very likely want to define our own functional form here. I'm
-    also a bit unsure how this form was even obtained, so further work here is very
-    needed.
+    This uses a standard Arrhenius equation to calculate the impact of temperature.
+
+    This function takes temperatures in Celsius as inputs and converts them into Kelvin
+    for use in the Arrhenius equation. TODO - review this after we have decided how to
+    handle these conversions in general.
 
     Args:
-        soil_temp: soil temperature for each soil grid cell [degrees C]
-        temp_scalar_coefficient_1: Unclear exactly what this parameter is [degrees C]
-        temp_scalar_coefficient_2: Unclear exactly what this parameter is [unclear]
-        temp_scalar_coefficient_3: Unclear exactly what this parameter is [unclear]
-        temp_scalar_coefficient_4: Unclear exactly what this parameter is [unclear]
-        temp_scalar_reference_temp: Reference temperature for temperature scalar
-            [degrees C]
+        soil_temperature: The temperature of the soil [C]
+        activation_energy: Energy of activation [J mol^-1]
+        soil_temperature: The reference temperature of the Arrhenius equation [C]
+        gas_constant: The molar gas constant [J mol^-1 K^-1]
 
     Returns:
-        A scalar that captures the impact of soil temperature on process rates
+        A multiplicative factor capturing the effect of temperature on microbial rates
     """
 
-    # This expression is drawn from Abramoff et al. (2018)
-    numerator = temp_scalar_coefficient_2 + (
-        temp_scalar_coefficient_3 / np.pi
-    ) * np.arctan(np.pi * (soil_temp - temp_scalar_coefficient_1))
-
-    denominator = temp_scalar_coefficient_2 + (
-        temp_scalar_coefficient_3 / np.pi
-    ) * np.arctan(
-        np.pi
-        * temp_scalar_coefficient_4
-        * (temp_scalar_reference_temp - temp_scalar_coefficient_1)
+    return np.exp(
+        (-activation_energy / gas_constant)
+        * ((1 / (soil_temperature + 273.15)) - (1 / (reference_temperature + 273.15)))
     )
```

Comment:
I think `273.15` is a good candidate for a universal constant

Reply:
I have added a draft `CoreConsts`  over in #323, so we can start to move things in and work out how to pass them around.